lib/cadet_web/controllers/auth_controller.ex

```diff
@@ -84,8 +113,8 @@ defmodule CadetWeb.AuthController do
     end
 
     response(200, "OK", Schema.ref(:Tokens))
-    response(400, "Missing parameter(s)")
-    response(403, "Wrong login attributes")
+    response(400, "Missing or invalid parameter")
+    response(500, "Internal server error")
```

Comment:
Eh, this happens?

Reply:
Yep, IVLE responds with internal server error we send a wrong API key, and I think setting an invalid API key on our end is internal-server-error-esque.

Reply:
Fair enough